CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ Breaking changes:
     - `3_move` and `7_actions` replaced by `navigation` group
   - editor context menu group changes:
     - `2_cut_copy_paste` renamed to `9_cutcopypaste` group
+- [core] `PreferenceProvider` [#3247](https://github.com/theia-ide/theia/pull/3247)
```

Comment:
i would group them under `multi-root workspace support for preferences`. There are more incompatible changes to what is listed here.

Reply:
i updated the list. could you check if there are items missing?